distribution/pom.xml

```diff
@@ -133,4 +137,58 @@
     </plugins>
   </build>
 
+  <profiles>
+    <profile>
+      <id>docker</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>io.fabric8</groupId>
+            <artifactId>docker-maven-plugin</artifactId>
+            <version>0.31.0</version>
+
+            <executions>
+              <execution>
+                <phase>install</phase>
+                <goals>
+                  <goal>build</goal>
+                </goals>
+              </execution>
+            </executions>
+
+            <configuration>
+              <images>
+                <image>
+                  <alias>styxcore</alias>
+                  <name>${styxcore.docker.image}</name>
+                  <build>
+                    <dockerFile>${project.basedir}/../docker/styx-image/Dockerfile</dockerFile>
```

Comment:
we probably can reference the parent project here instead of using  ${project.basedir}/../

Reply:
I have changed this to `${project.parent.basedir}`.